### PR TITLE
Major edits

### PR DESCRIPTION
**Pull request by resonant-gamedesign**

### CODE REVIEW

File: src/routes/spirit-board/growth.svelte [hunk 1]
```diff
@@ -7,6 +7,20 @@
...
+			while (spiritBoard.growth.growthSets[i].growthGroups.length > 0) {
+				firstSet.growthGroups.push(spiritBoard.growth.growthSets[i].growthGroups.shift());
+				firstSet.growthGroups[firstSet.growthGroups.length-1].id = firstSet.growthGroups.length;
+			}
+			spiritBoard = spiritBoard;
+		}
+		while(spiritBoard.growth.growthSets.length>1){
+		console.log(spiritBoard)
+		console.log('poppin')
+			spiritBoard.growth.growthSets.pop();
```
[the remark this answers is not in the document]
Comment (resonant-gamedesign) [reply src/routes/spirit-board/growth.svelte:22]: I'm saving all the growth groups and dumping them into the first growth set. Perhaps the function should be called 'stop using growth sets'.

File: src/routes/spirit-board/presence-tracks.svelte [hunk 2]
```diff
@@ -14,14 +45,59 @@
...
+									id={`energy${i}`}
+									class="input is-small"
+									type="text"
+									bind:value={spiritBoard.presenceTrack.energyNodes[i].effect}
+								/>
+							</div>
+						{/each}
 					</div>
 				</div>
+<!-- 				{#if spiritBoard.presenceTrack.useMiddleNodes}
```
[the remark this answers is not in the document]
Comment (resonant-gamedesign) [reply src/routes/spirit-board/presence-tracks.svelte]: No I was testing to remove it. We can remove it properly. I think I have.

File: static/template/My Custom Content/My Spirit/board_front_website.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (resonant-gamedesign) [reply static/template/My Custom Content/My Spirit/board_front_website.html]: For the final product, I haven't decided if folks should start with the empty template or an example or be able to load either. This empty template is helpful for testing (are the 'official spirit' templates I've added).

File: src/routes/spirit-board/index.svelte [hunk 4]
```diff
@@ -66,41 +110,456 @@
...
 
 	function setBoardValues(spiritBoard) {
 		if (frame) {
+			//Set Spirit Name and Image
 			const spiritName = frame.contentDocument.querySelectorAll('spirit-name')[0]
 			if (spiritName) {
 				spiritName.textContent = spiritBoard.nameAndArt.name
 			}
+
+			//Set Special Rules
```
[the remark this answers is not in the document]
Comment (resonant-gamedesign) [reply src/routes/spirit-board/index.svelte:124]: I don't think they'll ever be used on their own, but we could certainly break them up if you think its better that way. And then another function that just calls all of them at once.

File: src/routes/spirit-board/growth.svelte [hunk 5]
```diff
@@ -111,13 +153,14 @@
 									<div class="growth-group-title">
 										<div class="label">Growth Group
```
[the remark this answers is not in the document]
Comment (resonant-gamedesign) [reply src/routes/spirit-board/growth.svelte]: Anything that is <div class="label"... should instead by <label class="label"?

File: src/routes/spirit-board/growth.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (resonant-gamedesign) [reply src/routes/spirit-board/growth.svelte:22]: oooooh the final loop. Its removing all but the first growthSet, which now contains all the goodies we want to keep.
length>1 not length>0

File: src/routes/spirit-board/index.svelte (hunk 4 above)
[the remark this answers is not in the document]
Comment (neubee) [reply src/routes/spirit-board/index.svelte:124]: Try it out and see what you think. You would just take all the lines that make up the `//Set Special Rules` section (for example) and move them into a new function called something like `setSpecialRules(spiritBoard)` and then pass spiritBoard to it. Then call that function here.

File: src/routes/spirit-board/growth.svelte (hunk 5 above)
[the remark this answers is not in the document]
Comment (neubee) [reply src/routes/spirit-board/growth.svelte]: yep

[the remark this answers is not in the document]
Comment (neubee) [reply src/routes/spirit-board/growth.svelte]: Hopefully it doesn't break how things look.

File: src/routes/spirit-board/growth.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (neubee) [reply src/routes/spirit-board/growth.svelte:22]: Ah, ok. Thanks for clarifying that.

**Review by neubee — APPROVED** [review]

**Pull request merged**